### ISSUE TITLE
Menu for network doesn't parse ssid spaces correctly/ has duplicates

### ISSUE THREAD

**Issue by HerbFargus** [Death259/PiAssist#23]
The line of code here is what I'm referring to
https://github.com/Death259/PiAssist/blob/master/PiAssist.sh#L68

I think there should only be one print $2? 

Also if the SSID has spaces it splits it- so for example if my network is called "my special network" once it goes into the whiptail menu I have 3 networks called "my", "special",  and "network". 
If I run it through menu 2 on the scanner, it shows up as it should, but the troubles come once it is placed into the menu in option 3.

but if I just run the above line of code manually in the terminal it gives me the right list- so it must have something to do with whiptail. perhaps related to something like this? http://stackoverflow.com/questions/16897697/bash-spaces-in-whiptail-dialog-menu-items

**Comment by Death259** [comment]
Thanks for adding an issue on this. I had noticed it the other day, but forgot to turn it into an actionable item. I'll take look at what you mentioned and try that. If that doesn't work, I think the link you found is perfect. I will try to take a look tonight and see if I can fix it.

**Comment by Death259** [comment]
Yeah... this is a nightmare... can't seem to find a fix. I've posted a question on StackOverflow and hopefully someone there can find my mistake on why spaces in SSID names aren't being handled properly.

**Comment by HerbFargus** [comment]
yeah im pretty sure its an issue with whiptail rather than awk since it works just fine without the gui.

And we haven't even gotten to parsing the rest of the columns yet ;)

**Comment by Death259** [comment]
Yeah, I agree, it's gotta be a whiptail thing I'm missing. I'm sure it's just some quotes somewhere, but no matter what I tried, I couldn't get it to work...

To answer your question earlier on shouldn't there be only one print $2...

The reason there are 2, is because the first one is the "tag". Typically in the menus used throughout the PiAssist system there is a number in the front and then the actual menu item text people are choosing. When you pull the entry that the user selected we get the "tag" or in the case of most menus we get the menu item number. Since we know what the numbers represent when the script is being programmed, we can accommodate for those and not have to get fancy like we do with SSIDs. With the SSIDs, we don't know what they will be, so we can have the SSID be the "tag" as well as the actual menu entry text displayed. On the whiptail menu, you will see `--notags`. This hides the first occurrence of $2 better known as the SSID. When a user selects an SSID we grab the "tag" and save it to the variable that we reference later.

There may be a much better way accomplish this, but it's what I came up with at the time.
